src/mount/conf.cc

```diff
@@ -66,26 +66,20 @@ extern "C" void mount_ceph_get_config_info(const char *config_file,
 	continue;
     }
 
-    std::string addr;
-    addr += eaddr.ip_only_to_str();
-    addr += ":";
-    addr += std::to_string(eaddr.get_port());
-    /* If this will overrun cci_mons, stop here */
-    if (monaddrs.length() + 1 + addr.length() + 1 > sizeof(cci->cci_mons))
-      break;
-
-    if (first)
-      first = false;
-    else
-      monaddrs += ",";
-
-    monaddrs += addr;
+    if (oss.tellp() > 0) {
+      oss << ",";
+    }
+    oss << eaddr.get_sockaddr();
   }
 
-  if (monaddrs.length())
+  monaddrs = oss.str();
+  if (monaddrs.length() > sizeof(cci->cci_mons)) {
```

Comment:
What about space for `NUL` terminator?  The check that is being removed seemed to account for it.

Reply:
Good catch!